src/GafferScene/EvaluateLightLinks.cpp

```diff
@@ -49,6 +49,24 @@ using namespace GafferScene;
 IE_CORE_DEFINERUNTIMETYPED( EvaluateLightLinks );
 
 IECore::InternedString m_lightLinkAttrName = "linkedLights";
+IECore::InternedString m_shadowLinkAttrName = "linkedShadows";
+
+namespace{
+
+IECore::StringVectorDataPtr expressionToLightNames( ConstStringDataPtr expression, const ScenePlug *in )
```

Comment:
This function doesn't take ownership of `expression`, so `Ptr` isn't necessary. `const string &` is probably more appropriate, since it doesn't need `Data` anyway.